app/views/layouts/application.html.erb

```diff
@@ -89,12 +89,12 @@
 
 	<div id="global-app-error" class="app-error hidden"></div>
 
-	<script src="//ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
+	<script src="//ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```

Comment:
Shouldn't including jquery in the [application.js](https://github.com/ministryofjustice/courtfinder/pull/337/files#diff-a9c3bd311eab80c9ebe6a69830f9ad02R13) be enough? Do we need this line?